modules/n1ql/pages/n1ql-language-reference/typefun.adoc

```diff
@@ -1,36 +1,225 @@
 = Type Functions
-:description: Type functions perform operations that check or convert expressions.
+:description: The following checks if the supplied expression is a (data type).
```

Comment:
My bad, I meant use this text as the basis for the **Description** section in each of the type-checking functions.

```suggestion
:description: Type functions perform operations that check or convert expressions.
```